code-challenges.test.js

```diff
@@ -1,23 +1,64 @@
-// ASSESSMENT 3: Coding Practical Questions with Jest
-
-// Please read all questions thoroughly
-// Pseudo coding is REQUIRED
-// If you get stuck, please leave comments to help us understand your thought process
-
-// Use test driven development to complete the following questions
-// Add appropriate dependencies: $ yarn add jest
-
-// Reminder: The test will call your function
-// Run the file with the following command: $ yarn jest
-
-// --------------------1) Create a function that takes in a number (greater than 2) and returns an array that length containing the numbers of the Fibonacci sequence.
-
-// a) Create a test with expect statements for each of the variables provided.
-
-const fibLength1 = 6
-// Expected output: [1, 1, 2, 3, 5, 8]
-
-const fibLength2 = 10
+// // ASSESSMENT 3: Coding Practical Questions with Jest
+
+// const { exportAllDeclaration, exportNamedDeclaration, objectExpression } = require("@babel/types")
+
+// // Please read all questions thoroughly
+// // Pseudo coding is REQUIRED
+// // If you get stuck, please leave comments to help us understand your thought process
+
+// // Use test driven development to complete the following questions
+// // Add appropriate dependencies: $ yarn add jest
+
+// // Reminder: The test will call your function
+// // Run the file with the following command: $ yarn jest
+
+// // --------------------1) Create a function that takes in a number (greater than 2) and returns an array that length containing the numbers of the Fibonacci sequence.
+
+// // a) Create a test with expect statements for each of the variables provided.
+// //pseudo code: create a function that takes in a variable and returns an array the length of the variable containing the numbers of the Fibonacci sequence.(Each number is the sum of the two numbers before it.)
+
+// // describe ("fibonancciSequence",() =>{
+// //   it ("returns an array containing the numbers of the Fibonaci sequence", ()=>{
+// //     const fibLength1 = 6
```

Comment:
Nice job hoisting your variables inside of your tests, it is a really good practice!